src/test/suite/explorer/databaseTreeItem.test.ts

```diff
@@ -26,4 +26,35 @@ suite('DatabaseTreeItem Test Suite', () => {
       assert(collections[1].label === mockDatabases[mockDatabaseNames[1]].collections[1].name, `Expected a tree item child with the label collection name ${mockDatabases[mockDatabaseNames[1]].collections[1].name} found ${collections[1].label}`);
     }).then(() => done(), done);
```

Comment:
You can also write it like this:

`.then(done).catch(done);`

It might be easier to read and notice that done should happen both in case of the success of test and failure.

Reply:
Hmm I agree `.catch(done)` is a lot easier to read and declarative. eslint doesn't like it however since it's not a variable on `Thenable<void>`. Which is what a lot of the vscode interface functions use as a return type. I went and made the tests use `.then(done, done);` to be a bit cleaner. 🤔

Reply:
Let's do `done, done` then :)